src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitations/GetInvitationsQueryHandlerTests.cs

```diff
@@ -0,0 +1,582 @@
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 2);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnPunchOutDate()
+        {
+            var filter = new Filter { PunchOutDateFromUtc = new DateTime(2019, 10, 20, 10, 0, 0, DateTimeKind.Utc) };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 1);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnPersonOid()
+        {
+            var filter = new Filter { PersonOid = _frPersonGuid2 };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 1);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnFunctionalRoleCode()
+        {
+            var filter = new Filter { FunctionalRoleCode = _functionalRoleCode2 };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 1);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnCommPkgNoStartWith()
+        {
+            var filter = new Filter { CommPkgNoStartsWith = _commPkgNo2 };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 1);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnCommPkgNoStartWith2()
```

Comment:
rename Handler_ShouldFilterOnCommPkgNoStartWith2 .. I guess you test that the commpkg is found via McPkg and the testname should highlight so (if I'm right)